test/bonjour.js

```diff
@@ -136,8 +136,10 @@ test('bonjour.find - all services', function (bonjour, t) {
         t.equal(found[1], 'Foo Bar')
         t.equal(found[2], 'Invalid')
         t.equal(found[3], 'Sub Foo')
-        bonjour.destroy()
-        t.end()
+        setTimeout(function () {
```

Comment:
And what is the magic value `50`? Is there some way we can guarantee a safe time to end, rather than having a race condition (that we're just more likely to win)